full/atmos_ocean_fluxes_calc.f90

```diff
@@ -117,7 +117,7 @@ subroutine atmos_ocean_fluxes_calc(gas_fields_atm, gas_fields_ice,&
                     & gas_fields_atm%bc(n)%field(ind_u10)%values(i) /&
                     & (770.+45.*gas_fluxes%bc(n)%param(1)**(1./3.)) *&
                     & 101325./(rdgas*wtmair*1e-3*tsurf(i) *&
-                    & gas_fields_ice%bc(n)%field(ind_alpha)%values(i))
+                    & max(gas_fields_ice%bc(n)%field(ind_alpha)%values(i),epsln))
```

Comment:
@GFDL-Eric is there a guarantee that `gas_fields_ice%bc(n)%field(ind_alpha)%values(i)` is always greater than or equal to zero?

Reply:
I don't believe there's a check for non-negative values anywhere, on the other hand I'm not quite sure what a concentration value less than zero would physically represent.